components/ContentFingerprintLink.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <Link :href="href">
+    {{ item }}
+  </Link>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator'
+
+export enum ContentFirgerprints {
+  arweave = 'ar',
+  ipfs = 'ipfs',
+}
+
+@Component
+export default class ContentFingerprintLink extends Vue {
+  @Prop(String) readonly item: string | undefined
+
+  get prefix() {
+    return this.item && this.item.substr(0, this.item.indexOf(':'))
+  }
+
+  get href() {
+    if (this.item) {
+      switch (this.prefix) {
+        case ContentFirgerprints.arweave:
+          return `https://arweave.net/${this.item.slice(5)}`
+
+        case ContentFirgerprints.ipfs:
+          return `https://cloudflare-ipfs.com/ipfs/${this.item.slice(7)}`
+
+        default:
+          return this.item
+      }
+    }
+    return null
+  }
```

Comment:
Seems prefix can guard
```suggestion
  get href() {
    switch (this.prefix) {
      case ContentFirgerprints.arweave:
        return `https://arweave.net/${this.item.slice(5)}`

      case ContentFirgerprints.ipfs:
        return `https://cloudflare-ipfs.com/ipfs/${this.item.slice(7)}`

      default:
        return this.item
    }
  }
```